### ISSUE TITLE
Set up site

### ISSUE THREAD

**Issue by chasenlehara** [bit-docs/bit-docs-website#2]
Set up this repo with a very basic `bit-docs` site that has a home page with a brief description of the project and two pages in the sidebar nav: Guides and API.

**Comment by leoj3n** [comment]
I'm glad that this org will have an obvious name for the repo containing the website!

Unlike these somewhat confusingly named repos that contain their respective org websites...

DoneJS:

- https://github.com/donejs/donejs-next
- https://github.com/donejs/donejs/tree/gh-pages

StealJS:

- https://github.com/stealjs/stealjs
- https://github.com/stealjs/stealjs/tree/gh-pages

CanJS:

- https://github.com/canjs/canjs/tree/gh-pages

**Comment by chasenlehara** [comment]
@leoj3n, all of those use [GitHub pages](https://pages.github.com) and we’ve chosen to use the `gh-pages` branch for the published website instead of littering `master`, which is what I would expect us to do for this repo as well.

**Comment by justinbmeyer** [comment]
To reiterate chasen's point, gh-pages is expected for those familiar with github.

**Comment by leoj3n** [comment]
I didn't expect the following reply to turn into a manifesto, but I suppose that I do have some strongly held beliefs and feelings on this subject. I understand there is a risk that I may be missing a mark, but I figure it's better to be passionate rather than indifferent. At the end of the day, these are just my personally held beliefs from my experiences, and not written in stone or anything. So, take 'em at face value, if you have the time to read them. They may or may not be valuable to you:

I totally get where you guys are coming from, but I do feel the need to acknowledge my personal struggle w/ this!

When coming into the various orgs for the first time, as a young grasshopper excited about DoneJS, CanJS, StealJS, etc (however many months ago), I was familiar with `gh-pages`, but not bit-docs (or, at the time, DocumentJS/CSS). There were a number of barriers to understanding how each org's website (sausage) was being made, and even to this day [I still get bitten](https://github.com/donejs/donejs-next/pull/1#issuecomment-289537295) (I realize this may not be the most potent example, but does exemplify how easy it is to be in the wrong place).

The primary barriers are, 1) You have to locate the repo where the `gh-pages` branch lives (or `master` if it's setup like DoneJS-next), and then 2) Understand how the website is getting generated (which is different across orgs, can be a complex set of rules, and for the most part not well documented in the repo you are updating).

It is particularly confusing having three (or more, in certain cases) places where you might think to edit the same string:

- Markdown or code comments in the `master` branch.
  - Generated files, confusingly tracked in the `master` branch.
- Generated files tracked in the `gh-pages` branch.
  - Markdown or code comment files duplicated over from `master` into `gh-pages`.
- Markdown or code comments in _another_ repo entirely.
  - This case is rarely documented in the website repo's README.
  - Would not be obvious to someone without bit-docs experience.

Obviously, these gotchas won't confuse those yodas who set the system up, but I'm looking out for those other young grasshoppers with no bit-docs (and perhaps `gh-pages`) experience. The barriers are additionally compounded by the variances in documentation build strategies across orgs.

After banging up against these barriers enough times, I'm no longer confused like when I first began looking at these orgs. However, I do remember these being big barriers of entry for me when I wanted to start contributing in that way I thought would be the most logically easy place to start: Making small improvements to the guides I was reading on website.

I quickly found out I was in over my head!

TL;DR

All I'm saying is, I like how much more obvious it will be to people coming in for their first time to identify that this, indeed, is the website repo. They can know they are looking in the right place, and from there begin understanding how bit-docs pulls everything together because it will be stated clearly in the README, so that they can make their first contribution of a typo fix to the website or whatever (thus, hopefully turning what would be a 3-hour adventure into a quick 3-minute fix, and also increase the chances of them starting contributing in other ways).

Need to make it easy for people to contribute, 'specially in the way they are most likely to try first!

This _is_ great software, and although the contributing experience could be more streamlined in this respect for other repos, I do understand the need to respect the history of these projects, and don't expect for big immediate changes; but I do hope to influence what I can by my own example, specifically thinking of the opportunity to do so in a new project like bit-docs.

At the end of the day, patient people can get up and running with what's there now, perhaps after asking a question or two (which we're good about answering), and that's what's most important.

**Comment by chasenlehara** [comment]
@leoj3n, a couple things:

1. I think Guides should go first, similar to how the CanJS site is set up.
2. I think the second section should be named API, not Plugins, because it contains things that aren’t plugins (e.g. `bit-docs` itself).

**Comment by leoj3n** [comment]
@chasenlehara Made your suggested changes on that branch:

https://github.com/bit-docs/bit-docs-website/tree/package-json-api

**Comment by chasenlehara** [comment]
I’m closing this because the site was set up here: https://bit-docs.github.io/bit-docs-website/
